product_docs/docs/net_connector/8.0.2.1/04_installing_and_configuring_the_net_connector.mdx

```diff
@@ -249,10 +249,9 @@ Depending on your application type, you might need to import the namespace into
 
 ##### .NET Entity Framework Core
 
-To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is either:
+To configure the .NET Connector for use with Entity Framework Core, the data provider installation path is:
```

Comment:
This said "either" but only has one option

Reply:
yes it has only option now